.github/workflows/ci-casper-client-sdk.yml

```diff
@@ -47,52 +47,64 @@ jobs:
         uses: actions/setup-node@5b949b50c3461bbcd5a540b150c368278160234a #v3.4.0
         with:
           node-version: ${{ matrix.node-version }}
-          cache: 'npm'
+          cache: 'yarn'
+
+        # https://github.com/dtinth/setup-github-actions-caching-for-turbo
+      - name: Setup Turbo cache server
+        uses: dtinth/setup-github-actions-caching-for-turbo@e26dc6b0473cc072107f6cb4f6e69d33ac2fc386 # v1.1.0
```

Comment:
Do we have to rely on a commit dependency? It seems that there is a 1.2.0 tag in the repo for this plugin?

Reply:
we should use commit in the GitHub workflow, this is non-written rule.
https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash

Reply:
👍